include/z64scene.h

```diff
@@ -522,8 +522,8 @@ typedef enum {
 #define SCENE_CMD_ENTRANCE_LIST(entranceList) \
     { SCENE_CMD_ID_ENTRANCE_LIST, 0, CMD_PTR(entranceList) }
 
-#define SCENE_CMD_SPECIAL_FILES(elfMessageFile, keepObjectId) \
```

Comment:
Think I prefer `naviMessageFileId` here

Reply:
Id -> Index ? may be better than Num

and I don't think it needs to be so detailed about the implementation using files, the data may as well just be two arrays in code

Reply:
I suggested "Index" instead of "Id" since the values are currently raw literals, but "Id" would work if we add an enum